### ISSUE TITLE
Copy as HTML generates CSS with global effect

### ISSUE THREAD

**Issue by mscha** [mintty/mintty#889]
The HTML generated by Copy as HTML (all variations, even Copy as HTML text) generates CSS with global consequences.  Most annoyingly, it sets the margins and padding of `body` to 0.
The effect is that if you paste this somewhere (e.g. in a compose window in Thunderbird), it affects the entire message.

Can you please ensure the CSS doesn't have global effect?
The obvious way to do that, is to give the `div` containing the text a `class="mintty"` or similar, and ensure all CSS rules apply to (children of) `.mintty`.

**Comment by mintty** [comment]
Tweaked in repository.
Can you evaluate the current HTML creation?

**Comment by mscha** [comment]
I'm not able to make a build at this time, but looking at the commit, that only fixes my example annoyance, not the whole issue.
For instance, if you paste the generated HTML in an existing document, all existing `<pre>`s will lose their margin and padding, and every single `<span>` will suddenly have a fixed-width font.

My preferred solution is to put a `class="mintty"` on the main `<div>`, and make sure all CSS is only applied to children of `.mintty`.  For instance:

```
.mintty, pre.mintty { margin: 0; padding: 0 }
span.mintty {
  font-family: 'InputMono', 'Lucida Console ', 'Consolas', monospace;
}
```

and so on.
(Although it might be better to clean up the HTML/CSS a bit as well: for instance, it makes no sense to apply the fonts to the `span`, better to apply it to the `,mintty` class.)

**Comment by mintty** [comment]
Thanks, I'll give it another tweak.
However, applying formatting locally is on purpose, to my own annoyance, because some tools that you'd like to paste into don't apply the format otherwise, particularly PowerPoint.

**Comment by mintty** [comment]
Actually, `div` has `id='vt100'` already. I guess I'll just prefix `div` and `pre` with `#vt100 ` and add a `mintty` class to `body`.

**Comment by mintty** [comment]
Released 3.0.2.
